pos_analytic_by_config/readme/ROADMAP.rst

```diff
@@ -1 +1,5 @@
 * Add `account_analytic_id` in `pos.order` so we can use it in reports.
+* In `13.0` the session reconciliation has been refactored and thus the journal
```

Comment:
I remember a way that you can still have the generated journal entries "unpacked" in this version.

Reply:
I didn't see anything in the code. This is the only stuff that makes it from the line: https://github.com/odoo/odoo/blob/13.0/addons/point_of_sale/models/pos_session.py#L698-L704

Reply:
OK, I see.